### ISSUE TITLE
read_qc module

### ISSUE THREAD

**Issue by zhouyunyan** [bxlab/metaWRAP#107]
Hi, does the read_qc module include barcode removal?

**Comment by ursky** [comment]
Yes, trim_galore (part of the read_qc module) automatically determines the barcodes and trims them off.

**Comment by zhouyunyan** [comment]
If the company have trimed barcodes and  filtered out low quality sequences,  but it doesn't remove the host. Can I process it as raw data?
metawrap read_qc -1 RAW_READS/ERR011348_1.fastq -2 RAW_READS/ERR011348_2.fastq -t 24 -o READ_QC/ERR011348

**Comment by ursky** [comment]
Yes, you can process them as raw data to further remove over-represented kmers, trim low-quality ends, and remove human contamination.

**Comment by zhouyunyan** [comment]
ok. i have another question. why host contamination only for human,the intestinal sample of pigs or other animals? I think the types of host as an adjustable parameter would be better.

**Comment by ursky** [comment]
Good point. I implemented the change here and github to allow using a custom database prefix with the `-x` option. I also updated the database instructions. Can you replace your `miniconda/bin/metawrap-modules/read_qc` with the new one and see how it works for you? If it works ok, I will include the change in `metawrap v1.1.2`.
